BaseFilter/sections/specific.txt

```diff
@@ -343,6 +343,7 @@ $xmlhttprequest,third-party,denyallow=linkvertise.com|fastly.net|statically.io|s
 !
 ! NOTE: Specific rules
 !
+gptchatly.com##div.container > aside
```

Comment:
I'd prefer
`gptchatly.com##div[id^="desktop-sidebar-"]`

Reply:
There's also a mobile layout. Adds shows in the same aside elements
https://uploads.adguard.org/img_h7h6r.png

Reply:
Ok, your rule looks too wide, but it seems to make sense for this particular site.